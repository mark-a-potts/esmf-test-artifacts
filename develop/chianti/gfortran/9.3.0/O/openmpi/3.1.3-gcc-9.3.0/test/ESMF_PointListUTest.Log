build time -- 2021-04-23 11:58:54.445068
20210423 120417.128 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 120417.128 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210423 120417.128 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210423 120417.128 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210423 120417.128 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210423 120417.128 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 120417.128 INFO             PET0 Running with ESMF Version   : ESMF_8_2_0_beta_snapshot_05-2-g3f3d15a0c7
20210423 120417.128 INFO             PET0 ESMF library build date/time: "Apr 23 2021" "11:58:21"
20210423 120417.128 INFO             PET0 ESMF library build location : /home/mpotts/gfortran_9.3.0_openmpi_O_develop
20210423 120417.128 INFO             PET0 ESMF_COMM                   : openmpi
20210423 120417.128 INFO             PET0 ESMF_MOAB                   : enabled
20210423 120417.128 INFO             PET0 ESMF_LAPACK                 : enabled
20210423 120417.128 INFO             PET0 ESMF_NETCDF                 : enabled
20210423 120417.128 INFO             PET0 ESMF_PNETCDF                : disabled
20210423 120417.128 INFO             PET0 ESMF_PIO                    : enabled
20210423 120417.128 INFO             PET0 ESMF_YAMLCPP                : enabled
20210423 120417.129 INFO             PET0  Beginning Test, file ESMF_PointListUTest.F90, line 67
20210423 120417.129 INFO             PET0  NUMBER_OF_PROCESSORS           4
20210423 120417.129 INFO             PET0  PASS  Test creating a PointList from user inputs, ESMF_PointListUTest.F90, line 83
20210423 120417.129 INFO             PET0  PASS  Test creating a PointList from existing grid without masking, ESMF_PointListUTest.F90, line 98
20210423 120417.129 INFO             PET0  PASS  Test creating a PointList from existing grid with masking, ESMF_PointListUTest.F90, line 114
20210423 120417.129 INFO             PET0  PASS  Test creating a PointList from existing mesh via nodes without masking, ESMF_PointListUTest.F90, line 130
20210423 120417.130 INFO             PET0  PASS  Test creating a PointList from existing mesh via nodes with masking, ESMF_PointListUTest.F90, line 145
20210423 120417.130 INFO             PET0  PASS  Test creating a PointList from existing mesh via elements without masking, ESMF_PointListUTest.F90, line 160
20210423 120417.130 INFO             PET0  PASS  Test creating a PointList from existing mesh via elements with masking, ESMF_PointListUTest.F90, line 176
20210423 120417.130 INFO             PET0  PASS  Test creating a PointList from existing LocStream without masking, ESMF_PointListUTest.F90, line 191
20210423 120417.130 INFO             PET0  PASS  Test creating a PointList from existing LocStream with masking, ESMF_PointListUTest.F90, line 206
20210423 120417.131 INFO             PET0  PASS  Test creating a PointList from existing mesh via nodes without masking, empty proc, ESMF_PointListUTest.F90, line 221
20210423 120417.131 INFO             PET0  Ending Test, file ESMF_PointListUTest.F90, line 226
20210423 120417.131 INFO             PET0 Finalizing ESMF
20210423 120417.128 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 120417.128 INFO             PET1 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210423 120417.128 INFO             PET1 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210423 120417.128 INFO             PET1 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210423 120417.128 INFO             PET1 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210423 120417.128 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 120417.128 INFO             PET1 Running with ESMF Version   : ESMF_8_2_0_beta_snapshot_05-2-g3f3d15a0c7
20210423 120417.128 INFO             PET1 ESMF library build date/time: "Apr 23 2021" "11:58:21"
20210423 120417.128 INFO             PET1 ESMF library build location : /home/mpotts/gfortran_9.3.0_openmpi_O_develop
20210423 120417.128 INFO             PET1 ESMF_COMM                   : openmpi
20210423 120417.128 INFO             PET1 ESMF_MOAB                   : enabled
20210423 120417.128 INFO             PET1 ESMF_LAPACK                 : enabled
20210423 120417.128 INFO             PET1 ESMF_NETCDF                 : enabled
20210423 120417.128 INFO             PET1 ESMF_PNETCDF                : disabled
20210423 120417.128 INFO             PET1 ESMF_PIO                    : enabled
20210423 120417.128 INFO             PET1 ESMF_YAMLCPP                : enabled
20210423 120417.129 INFO             PET1  Beginning Test, file ESMF_PointListUTest.F90, line 67
20210423 120417.129 INFO             PET1  NUMBER_OF_PROCESSORS           4
20210423 120417.129 INFO             PET1  PASS  Test creating a PointList from user inputs, ESMF_PointListUTest.F90, line 83
20210423 120417.129 INFO             PET1  PASS  Test creating a PointList from existing grid without masking, ESMF_PointListUTest.F90, line 98
20210423 120417.129 INFO             PET1  PASS  Test creating a PointList from existing grid with masking, ESMF_PointListUTest.F90, line 114
20210423 120417.129 INFO             PET1  PASS  Test creating a PointList from existing mesh via nodes without masking, ESMF_PointListUTest.F90, line 130
20210423 120417.130 INFO             PET1  PASS  Test creating a PointList from existing mesh via nodes with masking, ESMF_PointListUTest.F90, line 145
20210423 120417.130 INFO             PET1  PASS  Test creating a PointList from existing mesh via elements without masking, ESMF_PointListUTest.F90, line 160
20210423 120417.130 INFO             PET1  PASS  Test creating a PointList from existing mesh via elements with masking, ESMF_PointListUTest.F90, line 176
20210423 120417.130 INFO             PET1  PASS  Test creating a PointList from existing LocStream without masking, ESMF_PointListUTest.F90, line 191
20210423 120417.130 INFO             PET1  PASS  Test creating a PointList from existing LocStream with masking, ESMF_PointListUTest.F90, line 206
20210423 120417.131 INFO             PET1  PASS  Test creating a PointList from existing mesh via nodes without masking, empty proc, ESMF_PointListUTest.F90, line 221
20210423 120417.131 INFO             PET1  Ending Test, file ESMF_PointListUTest.F90, line 226
20210423 120417.131 INFO             PET1 Finalizing ESMF
20210423 120417.128 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 120417.128 INFO             PET2 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210423 120417.128 INFO             PET2 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210423 120417.128 INFO             PET2 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210423 120417.128 INFO             PET2 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210423 120417.128 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 120417.128 INFO             PET2 Running with ESMF Version   : ESMF_8_2_0_beta_snapshot_05-2-g3f3d15a0c7
20210423 120417.128 INFO             PET2 ESMF library build date/time: "Apr 23 2021" "11:58:21"
20210423 120417.128 INFO             PET2 ESMF library build location : /home/mpotts/gfortran_9.3.0_openmpi_O_develop
20210423 120417.128 INFO             PET2 ESMF_COMM                   : openmpi
20210423 120417.128 INFO             PET2 ESMF_MOAB                   : enabled
20210423 120417.128 INFO             PET2 ESMF_LAPACK                 : enabled
20210423 120417.128 INFO             PET2 ESMF_NETCDF                 : enabled
20210423 120417.128 INFO             PET2 ESMF_PNETCDF                : disabled
20210423 120417.128 INFO             PET2 ESMF_PIO                    : enabled
20210423 120417.128 INFO             PET2 ESMF_YAMLCPP                : enabled
20210423 120417.129 INFO             PET2  Beginning Test, file ESMF_PointListUTest.F90, line 67
20210423 120417.129 INFO             PET2  NUMBER_OF_PROCESSORS           4
20210423 120417.129 INFO             PET2  PASS  Test creating a PointList from user inputs, ESMF_PointListUTest.F90, line 83
20210423 120417.129 INFO             PET2  PASS  Test creating a PointList from existing grid without masking, ESMF_PointListUTest.F90, line 98
20210423 120417.129 INFO             PET2  PASS  Test creating a PointList from existing grid with masking, ESMF_PointListUTest.F90, line 114
20210423 120417.129 INFO             PET2  PASS  Test creating a PointList from existing mesh via nodes without masking, ESMF_PointListUTest.F90, line 130
20210423 120417.130 INFO             PET2  PASS  Test creating a PointList from existing mesh via nodes with masking, ESMF_PointListUTest.F90, line 145
20210423 120417.130 INFO             PET2  PASS  Test creating a PointList from existing mesh via elements without masking, ESMF_PointListUTest.F90, line 160
20210423 120417.130 INFO             PET2  PASS  Test creating a PointList from existing mesh via elements with masking, ESMF_PointListUTest.F90, line 176
20210423 120417.130 INFO             PET2  PASS  Test creating a PointList from existing LocStream without masking, ESMF_PointListUTest.F90, line 191
20210423 120417.130 INFO             PET2  PASS  Test creating a PointList from existing LocStream with masking, ESMF_PointListUTest.F90, line 206
20210423 120417.131 INFO             PET2  PASS  Test creating a PointList from existing mesh via nodes without masking, empty proc, ESMF_PointListUTest.F90, line 221
20210423 120417.131 INFO             PET2  Ending Test, file ESMF_PointListUTest.F90, line 226
20210423 120417.131 INFO             PET2 Finalizing ESMF
20210423 120417.128 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 120417.128 INFO             PET3 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210423 120417.128 INFO             PET3 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210423 120417.128 INFO             PET3 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210423 120417.128 INFO             PET3 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210423 120417.128 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 120417.128 INFO             PET3 Running with ESMF Version   : ESMF_8_2_0_beta_snapshot_05-2-g3f3d15a0c7
20210423 120417.128 INFO             PET3 ESMF library build date/time: "Apr 23 2021" "11:58:21"
20210423 120417.128 INFO             PET3 ESMF library build location : /home/mpotts/gfortran_9.3.0_openmpi_O_develop
20210423 120417.128 INFO             PET3 ESMF_COMM                   : openmpi
20210423 120417.128 INFO             PET3 ESMF_MOAB                   : enabled
20210423 120417.128 INFO             PET3 ESMF_LAPACK                 : enabled
20210423 120417.128 INFO             PET3 ESMF_NETCDF                 : enabled
20210423 120417.128 INFO             PET3 ESMF_PNETCDF                : disabled
20210423 120417.128 INFO             PET3 ESMF_PIO                    : enabled
20210423 120417.128 INFO             PET3 ESMF_YAMLCPP                : enabled
20210423 120417.129 INFO             PET3  Beginning Test, file ESMF_PointListUTest.F90, line 67
20210423 120417.129 INFO             PET3  NUMBER_OF_PROCESSORS           4
20210423 120417.129 INFO             PET3  PASS  Test creating a PointList from user inputs, ESMF_PointListUTest.F90, line 83
20210423 120417.129 INFO             PET3  PASS  Test creating a PointList from existing grid without masking, ESMF_PointListUTest.F90, line 98
20210423 120417.129 INFO             PET3  PASS  Test creating a PointList from existing grid with masking, ESMF_PointListUTest.F90, line 114
20210423 120417.129 INFO             PET3  PASS  Test creating a PointList from existing mesh via nodes without masking, ESMF_PointListUTest.F90, line 130
20210423 120417.130 INFO             PET3  PASS  Test creating a PointList from existing mesh via nodes with masking, ESMF_PointListUTest.F90, line 145
20210423 120417.130 INFO             PET3  PASS  Test creating a PointList from existing mesh via elements without masking, ESMF_PointListUTest.F90, line 160
20210423 120417.130 INFO             PET3  PASS  Test creating a PointList from existing mesh via elements with masking, ESMF_PointListUTest.F90, line 176
20210423 120417.130 INFO             PET3  PASS  Test creating a PointList from existing LocStream without masking, ESMF_PointListUTest.F90, line 191
20210423 120417.130 INFO             PET3  PASS  Test creating a PointList from existing LocStream with masking, ESMF_PointListUTest.F90, line 206
20210423 120417.131 INFO             PET3  PASS  Test creating a PointList from existing mesh via nodes without masking, empty proc, ESMF_PointListUTest.F90, line 221
20210423 120417.131 INFO             PET3  Ending Test, file ESMF_PointListUTest.F90, line 226
20210423 120417.131 INFO             PET3 Finalizing ESMF
