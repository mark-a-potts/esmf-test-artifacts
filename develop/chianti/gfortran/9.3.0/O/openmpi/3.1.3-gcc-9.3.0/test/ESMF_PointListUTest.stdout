build time -- 2021-04-23 11:58:54.445068
  Beginning Test, file ESMF_PointListUTest.F90, line 67
  NUMBER_OF_PROCESSORS           4
  Beginning Test, file ESMF_PointListUTest.F90, line 67
  NUMBER_OF_PROCESSORS           4
  Beginning Test, file ESMF_PointListUTest.F90, line 67
  NUMBER_OF_PROCESSORS           4
  Beginning Test, file ESMF_PointListUTest.F90, line 67
  NUMBER_OF_PROCESSORS           4
  PASS  Test creating a PointList from user inputs, ESMF_PointListUTest.F90, line 83
  PASS  Test creating a PointList from user inputs, ESMF_PointListUTest.F90, line 83
  PASS  Test creating a PointList from user inputs, ESMF_PointListUTest.F90, line 83
  PASS  Test creating a PointList from user inputs, ESMF_PointListUTest.F90, line 83
  PASS  Test creating a PointList from existing grid without masking, ESMF_PointListUTest.F90, line 98
  PASS  Test creating a PointList from existing grid without masking, ESMF_PointListUTest.F90, line 98
  PASS  Test creating a PointList from existing grid without masking, ESMF_PointListUTest.F90, line 98
  PASS  Test creating a PointList from existing grid without masking, ESMF_PointListUTest.F90, line 98
  PASS  Test creating a PointList from existing grid with masking, ESMF_PointListUTest.F90, line 114
  PASS  Test creating a PointList from existing grid with masking, ESMF_PointListUTest.F90, line 114
  PASS  Test creating a PointList from existing grid with masking, ESMF_PointListUTest.F90, line 114
  PASS  Test creating a PointList from existing grid with masking, ESMF_PointListUTest.F90, line 114
  PASS  Test creating a PointList from existing mesh via nodes without masking, ESMF_PointListUTest.F90, line 130
  PASS  Test creating a PointList from existing mesh via nodes without masking, ESMF_PointListUTest.F90, line 130
  PASS  Test creating a PointList from existing mesh via nodes without masking, ESMF_PointListUTest.F90, line 130
  PASS  Test creating a PointList from existing mesh via nodes without masking, ESMF_PointListUTest.F90, line 130
  PASS  Test creating a PointList from existing mesh via nodes with masking, ESMF_PointListUTest.F90, line 145
  PASS  Test creating a PointList from existing mesh via nodes with masking, ESMF_PointListUTest.F90, line 145
  PASS  Test creating a PointList from existing mesh via nodes with masking, ESMF_PointListUTest.F90, line 145
  PASS  Test creating a PointList from existing mesh via nodes with masking, ESMF_PointListUTest.F90, line 145
  PASS  Test creating a PointList from existing mesh via elements without masking, ESMF_PointListUTest.F90, line 160
  PASS  Test creating a PointList from existing mesh via elements without masking, ESMF_PointListUTest.F90, line 160
  PASS  Test creating a PointList from existing mesh via elements without masking, ESMF_PointListUTest.F90, line 160
  PASS  Test creating a PointList from existing mesh via elements without masking, ESMF_PointListUTest.F90, line 160
  PASS  Test creating a PointList from existing mesh via elements with masking, ESMF_PointListUTest.F90, line 176
  PASS  Test creating a PointList from existing mesh via elements with masking, ESMF_PointListUTest.F90, line 176
  PASS  Test creating a PointList from existing mesh via elements with masking, ESMF_PointListUTest.F90, line 176
  PASS  Test creating a PointList from existing mesh via elements with masking, ESMF_PointListUTest.F90, line 176
  PASS  Test creating a PointList from existing LocStream without masking, ESMF_PointListUTest.F90, line 191
  PASS  Test creating a PointList from existing LocStream without masking, ESMF_PointListUTest.F90, line 191
  PASS  Test creating a PointList from existing LocStream without masking, ESMF_PointListUTest.F90, line 191
  PASS  Test creating a PointList from existing LocStream without masking, ESMF_PointListUTest.F90, line 191
  PASS  Test creating a PointList from existing LocStream with masking, ESMF_PointListUTest.F90, line 206
  PASS  Test creating a PointList from existing LocStream with masking, ESMF_PointListUTest.F90, line 206
  PASS  Test creating a PointList from existing LocStream with masking, ESMF_PointListUTest.F90, line 206
  PASS  Test creating a PointList from existing LocStream with masking, ESMF_PointListUTest.F90, line 206
  PASS  Test creating a PointList from existing mesh via nodes without masking, empty proc, ESMF_PointListUTest.F90, line 221
  Ending Test, file ESMF_PointListUTest.F90, line 226
  PASS  Test creating a PointList from existing mesh via nodes without masking, empty proc, ESMF_PointListUTest.F90, line 221
  Ending Test, file ESMF_PointListUTest.F90, line 226
  PASS  Test creating a PointList from existing mesh via nodes without masking, empty proc, ESMF_PointListUTest.F90, line 221
  Ending Test, file ESMF_PointListUTest.F90, line 226
  PASS  Test creating a PointList from existing mesh via nodes without masking, empty proc, ESMF_PointListUTest.F90, line 221
  Ending Test, file ESMF_PointListUTest.F90, line 226
  PET           2  Test Elapsed Time    3.0939999999999994       msec.
  PET           0  Test Elapsed Time    3.0710000000000006       msec.
  PET           1  Test Elapsed Time    3.0290000000000004       msec.
  PET           3  Test Elapsed Time    3.0440000000000000       msec.
